lit/setup-and-operations/install.lit

```diff
@@ -68,6 +68,232 @@ page.
   different machine than the server.
 }
 
+
+\section{
+  \title{Configuring Auth Providers}{auth-config}
+
+  At startup, you will need to configure any local users or third-party auth
+  providers that you wish to use with Concourse.
+
+  In contrast to previous releases of Concourse, \code{users} are now central
+  to the authentication flow. Instead of logging in as a team, you now log in
+  as a user and can belong to one or more teams. Users can be added to a team
+  by configuring the team's whitelist as described in
+  \reference{authentication}.
+
+  \section{
+    \title{Local Auth}{local-auth-config}
+
+    In order to add new users to the system, you can supply the
+    \code{--add-local-user} flag to the \code{concourse web} subcommand like
+    so:
+
+    \codeblock{bash}{{{
+    concourse web \
+      --add-local-user myuser:mypass
+    }}}
```

Comment:
A paragraph here saying what this effectively does (or doesn't do) may be useful.

For example, "all this does is whitelist the username/password for logging in. Local users don't intrinsically have any capabilities - they must be granted access to individual teams before they can do anything". And then an example of granting access with `set-team` and `--main-team-local-user`. (Or link to the relevant section - I see this is covered already in the 'teams' section.)